### PR TITLE
Feature / Add loading state to camera app

### PR DESCRIPTION
**Pull request by jfrader**

### CODE REVIEW

**Comment by jfrader** [comment]
Needs more testing ^

**Review by itschip — APPROVED** [review]

**Comment by RockySouthpaw** [comment]
Is this just for pictures being sent?

**Comment by jfrader** [comment]
This is only for taking a new picture.

File: phone/src/apps/camera/components/grid/GalleryGrid.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { Box, Button } from '@material-ui/core';
+import { Box, Button, CircularProgress } from '@material-ui/core';
 import useStyles from './grid.styles';
-import { usePhotos } from '../../hooks/usePhotos';
+import { useCamera } from '../../hooks/useCamera';
```
Comment (itschip) [inline phone/src/apps/camera/components/grid/GalleryGrid.tsx:4]: Remember to change the import in GalleryModal and ShareModal. Those are still trying to import 'usePhotos'

**Review by itschip — CHANGES_REQUESTED** [review]

**Review by TasoOneAsia — APPROVED** [review]

**Review by itschip — APPROVED** [review]

**Pull request merged**